src/LiveComponent/src/EventListener/LiveComponentSubscriber.php

```diff
@@ -143,13 +151,14 @@ public function onKernelController(ControllerEvent $event): void
             throw new NotFoundHttpException(sprintf('The action "%s" either doesn\'t exist or is not allowed in "%s". Make sure it exist and has the LiveAction attribute above it.', $action, \get_class($component)));
         }
```

Comment:
This wouldn't catch the case where a batch request is sent with invalid action (because the request event is skipped for sub-requests). Or are you thinking for `_batch` to loop over and validate?